lib/tasks/incremental_update.rake

```diff
@@ -2,12 +2,14 @@
 namespace :blacklight do
   desc "Re-index Blacklight Catalog"
   task reindex: :environment do
-    oai_set = if Rails.env.casecmp('production').zero?
-                "blacklight"
-              else
-                "blacklighttest"
-              end
-    exec("/usr/local/bin/bundle exec rails marc_index_ingest oai_set_name=#{oai_set} | /usr/bin/logger -t blacklight_reindex")
-    Honeybadger.check_in(ENV['HONEYBADGER_BLACKLIGHT_REINDEX_CHECK_IN_ID']) if Rails.env.production?
```

Comment:
Once you run the `exec` method, you can no longer run any Ruby code after, which explains why the check ins were never processed.